src/core/qgstracer.h

```diff
@@ -61,10 +63,15 @@ class CORE_EXPORT QgsTracer : public QObject
     QgsCoordinateReferenceSystem destinationCrs() const { return mCRS; }
 
     /**
-     * Sets the \a crs and transform \a context used for tracing.
+     * Sets the \a crs and \a transformContext used for tracing.
      * \see destinationCrs()
      */
-    void setDestinationCrs( const QgsCoordinateReferenceSystem &crs, const QgsCoordinateTransformContext &context );
+    void setDestinationCrs( const QgsCoordinateReferenceSystem &crs, const QgsCoordinateTransformContext &transformContext );
```

Comment:
Isn't this an api break? (rename of context->transformContext).

Reply:
There's an outstanding issue on the API break issue tracker about this - maybe just add the script to that?